### PR TITLE
Add Watchdog RCA page

### PR DESCRIPTION
**Pull request by urseberry**
### Preview

https://docs-staging.datadoghq.com/uchen/watchdog-rca/watchdog/rca/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/watchdog/rca/_index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Watchdog never classifies degraded application performance, such as higher latency or new errors, as the root cause of an incident. Datadog calls an initial symptom of degraded application performance a **critical failure**, as described below. 
+
+### Critical failure
+
+The critical failure section highlights where and how the root cause first (and most directly) causes degraded application performance. Critical failures always include a latency or error rate increase.
+
+### Impact
+
+Watchdog RCA also identifies services indirectly affected by the root cause. Any performance degradation listed in Impact is expected to recover once the Critical Failure is resolved. For RUM users, Watchdog also automatically assesses which view paths and users were affected by the performance anomalies. [screenshot]
```
Comment (urseberry) [inline content/en/watchdog/rca/_index.md:50]: Brooke, do you still want to include a screenshot here?

**Review by urseberry — COMMENTED** [review]
Brooke, do you mind if I update the main Watchdog RCA screenshot to remove the side panel, and focus on just the Watchdog RCA section?

**Comment by urseberry** [comment]
After spending some time with the page, I agree that we should leave the main screenshot in place, because it helps the user understand the three main parts of RCA and provides examples of them.

**Review by urseberry — COMMENTED** [review]
I'm not sure why I made RCA into a directory. It can probably just be a file under the Watchdog directory. May, what do you think?

**Comment by urseberry** [comment]
> 

Moved the page out of the directory and into the main Watchdog foler.

**Comment by urseberry** [comment]
> I'm not sure why I made RCA into a directory. Maybe because I saw that `faq` is a directory? RCA can probably just be a file under the Watchdog directory. May, what do you think?

I removed the RCA directory and put the new page under the main Watchdog folder instead.

**Comment by maycmlee** [comment]
Hey @urseberry I think the RCA page under Watchdog works. We only need a separate RCA folder if we have RCA subpages.

File: content/en/watchdog/rca.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+title: Watchdog RCA
+kind: documentation
+further_reading:
+- link: "/watchdog/faq/root-cause-not-showing/"
+  tag: "Documentation"
+  text: "Root cause not showing"
+---
```
Comment (maycmlee) [inline content/en/watchdog/rca.md:8]: What about including this blog? https://www.datadoghq.com/blog/datadog-watchdog-automated-root-cause-analysis/

**Comment by crunchypistachiobutter** [comment]
LGTM

File: content/en/watchdog/rca.md (hunk 2 above)
Comment (crunchypistachiobutter) [reply content/en/watchdog/rca.md:11]: this blog post will be replaced soon

**Review by maycmlee — APPROVED** [review]
Just a small edit! Probably reverting something I had changed earlier. Otherwise, looks good to me.

**Pull request merged**